Cargo.toml

```diff
@@ -21,3 +21,6 @@ neon = { git = 'https://github.com/signalapp/neon', tag = 'signal-2021-06-02' }
 
 [profile.dev.package.num-bigint-dig]
 opt-level = 2 # too slow otherwise!
+
+[profile.release]
+panic = "abort"
```

Comment:
We're deliberately recovering from panics today; since most of the types here don't expose mutating operations, it's not hard to treat a panic like any other error (albeit one that should never happen). Are the new architectures missing that part of the Rust stdlib infrastructure?

Reply:
They fail to compile without this change, yeah. I don't fully understand the ramifications

Reply:
I figured this out (we have to build panic_unwind from source with xargo too)